functions/helloworld/hello-pubsub/pom.xml

```diff
@@ -145,6 +145,7 @@
         <version>0.9.7</version>
         <configuration>
           <functionTarget>functions.HelloPubSub</functionTarget>
+          <port>8083</port>
```

Comment:
Should we be hard coding this value, even if it's for the tests? Does this change how a user will deploy this sample?

Reply:
It shouldn't change deployment or test-running.

(cl/377974756 is the only thing changed by this PR.)

Reply:
I see, can you add a comment for the user on why it's there

Reply:
I added one to the sample itself (already).

Bearing that in mind, did you *also* want me to add something to `pom.xml`?

Reply:
Ah I see. lgtm